src/TraceEvent/Symbols/SymbolReader.cs

```diff
@@ -1901,11 +1912,21 @@ protected virtual string GetSourceFromSrcServer()
             if (url != null)
             {
                 HttpClient httpClient = new HttpClient();
```

Comment:
Since you're already in here, can you please put a using block around the construction of the ```HttpClient```?